user-guide.html

```diff
@@ -254,28 +261,31 @@ <h1 class="font-head">Our data</h1><br>
 
     <!-- Content block -->
     <div class="row" id="block4">
-      <div class="col-sm-2"></div><!-- Spacing col -->
+      <div class="col-xl-2"></div><!-- Spacing col -->
 
         <!-- Left content -->
-        <div class="col-sm mainContent">
+        <div class="col-xl-5 mainContent">
           <div class="content-container">
             <h1 class="font-head">Data layers</h1><br>
             <p>There are two types of data layers:</p>
               <ul>
-                <li><p><strong>Risk factors:</strong> Factors which indicate that a community may be at elevated risk during the COVID-19 pandemic</p></li>
-                <li><p><strong>Community resources:</strong> Factors which indicate that a community may have better resources for coping during COVID-19 pandemic</p></li>
+                <li><p><strong>Local need:</strong> Factors which indicate that a community may be at elevated risk during the COVID-19 pandemic</p></li>
+                <li><p><strong>Local support:</strong> Factors which indicate that a community may have better resources for coping during COVID-19 pandemic</p></li>
               </ul>
+            <p>We are using covid mutual aid database to identify community support groups, if you don't your local group on the map 
```

Comment:
Could I suggest a change to the wording here? 
"We are using the Police Rewired community support dataset (https://www.policecoders.org/home/covid-19) to map community support groups. If you don't see your local group then you can add it by registering on the Covid Mutual Aid website. (https://covidmutualaid.org/)"

Reply:
Perfect thank you that's much nicer wording - done.